ams/optimizer/flink-optimizer/pom.xml

```diff
@@ -177,6 +177,11 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Does the `flink-optimizer` module still need to declear the dependency as it had dependent on `amoro-core`?

Reply:
I will fix this.